storops/unity/resource/lun.py

```diff
@@ -86,7 +105,7 @@ def get_nested_properties(cls):
     def create(cls, cli, name, pool, size, sp=None, host_access=None,
                is_thin=None, description=None, io_limit_policy=None,
                is_repl_dst=None, tiering_policy=None, snap_schedule=None,
-               is_compression=None):
+               is_data_reduction_enabled=None):
```

Comment:
You shouldn't change the API directly. It breaks the back-port capability.
You could append the new parameter after `is_compression`, and make these two parameters the same.

Reply:
Was "is_compression" ever used somewhere? We didn't support it before, did we?

Reply:
Done, using the legacy parameter is_compression.

Reply:
@amagicsun No, we didn't support it in Openstack cinder driver, but maybe someone use storops directly, and call this function in their own codes.